lld/ELF/MarkLive.cpp

```diff
@@ -42,16 +44,30 @@ using namespace lld;
 using namespace lld::elf;
 
 namespace {
-template <class ELFT> class MarkLive {
+
```

Comment:
delete the blank line after `namespace {`

Reply:
Done.